tree/ntuple/v7/src/RPageSinkBuf.cxx

```diff
@@ -200,7 +206,10 @@ void ROOT::Experimental::Internal::RPageSinkBuf::CommitPage(ColumnHandle_t colum
       // Make sure the page buffer is not aliased so that we can free the uncompressed page.
       config.fAllowAlias = false;
       config.fBuffer = zipItem.fBuf.get();
-      sealedPage = SealPage(config);
+      {
+         Detail::RNTupleAtomicTimer timer(fCounters->fTimeWallZip, fCounters->fTimeCpuZip);
+         sealedPage = SealPage(config);
+      }
```

Comment:
Hm, at least for the wall time, this may be confusing because tasks can run in parallel.

Reply:
Right, reading also only counts the time in `RPageSourceFile::LoadPageImpl` when executed in a blocking fashion... I removed the time from the IMT code path for now (I'm mostly interested in the non-IMT case for parallel writing), but it would be great to somehow get observability into background (de)compression as well.